frontend/ui/widget/verticalscrollbar.lua

```diff
@@ -13,6 +13,7 @@ local VerticalScrollBar = Widget:new{
     bordercolor = Blitbuffer.COLOR_BLACK,
     radius = 0,
     rectcolor = Blitbuffer.COLOR_BLACK,
+    min_thumb_size = Size.line.thick,
```

Comment:
Thumb? Wouldn't min_scrollbar_width or something make more sense or am I misunderstanding it completely?

Reply:
It's the actual scrollbar 'notch', not the scrollbar itself. Which I somehow also forgot the usual name of :D.

Reply:
Well, I dunno how to name that little black area inside the full white inner scrollbar, that represents the current page size and position relative to the whole document.
When the document is big, and you're only displaying a little fraction of the document height (like in the screenshot above), the little black thing was sized 0 and not visible. So I decided to ensure a minimal size (actually height rather than width, so I named it "size" :| ).
I googled it and found "thumb" (may be in context where one could actually move it with the mouse or a finger...).
Any other name? notch then. or stick with thumb?
(Size.line.thick, cause it was the right size I needed :), the other Size.line were too small. Yeah, that's not a line, but it becomes a line when too small :) should I add something to Size.item ?)

Reply:
I thought that was called something like the grip or the knob but Gtk and Microsoft seem to call it the thumb.

https://en.wikipedia.org/wiki/Scrollbar
> The “thumb” has different names in different environments: on the Mac OS X 10.4 it is called a "scroller";[3] on the Java platform it is called "thumb" or "knob"; Microsoft's .NET documentation refers to it as "scroll box" or "scroll thumb"; in other environments it is called "elevator", "quint", "puck", "wiper" or "grip". 

Anyway, carry on then. Maybe add a comment saying what it is? Maybe it's just a weird gap in my knowledge. ;-)